.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,19 @@
+name: Release Drafter
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - master
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      # Drafts your next Release notes as Pull Requests are merged into "master"
+      - uses: release-drafter/release-drafter@v5.8.0
+        with:
+          # (Optional) specify config name to use, relative to .github/. Default: release-drafter.yml
+          # config-name: my-config.yml
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Not quite sure if we need to set a secret for this? I'll be in a meeting soon, will check back after.

Reply:
Seems to be set automatically for every Github Action, see https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token. All good then.